Demo/RCHSearchViewController.swift

```diff
@@ -86,14 +127,51 @@ extension RCHSearchViewController: UISearchBarDelegate, UICollectionViewDelegate
         footerView.addGestureRecognizer(tapRecognizer)
         autocompleteTableView.tableFooterView = footerView
         
+        pickerView.frame = CGRect(x: 0, y: view.frame.height - 300, width: view.frame.width, height: 300)
+        sortPicker.dataSource = self
+        sortPicker.delegate = self
+        sortPicker.frame = CGRect(x: 0, y: 30, width: view.frame.width, height: 270)
+        sortPicker.backgroundColor = UIColor.lightGray
+        sortPicker.showsSelectionIndicator = true
+        
         searchResultsCollectionView!.register(UINib(nibName: "RCHProductCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: reuseIdentifier)
         
-        configureAPI()
+        let toolBar = UIToolbar()
+        toolBar.frame = CGRect(x: 0, y: 0, width: view.frame.width, height: 30)
+        toolBar.barStyle = .default
+        toolBar.isTranslucent = true
+        toolBar.tintColor = UIColor.blue
+        toolBar.sizeToFit()
+        
+        let doneButton = UIBarButtonItem(title: "Done", style: .plain, target: self, action: #selector(self.sortPickerDoneSelected))
+        let spaceButton = UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: nil, action: nil)
+        let cancelButton = UIBarButtonItem(title: "Cancel", style: .plain, target: self, action: #selector(self.sortPickerCancelSelected))
+        
+        toolBar.setItems([cancelButton, spaceButton, doneButton], animated: false)
+        toolBar.isUserInteractionEnabled = true
+        
+        sortFilterView.isHidden = true
+        pickerView.addSubview(sortPicker)
+        pickerView.addSubview(toolBar)
+        view.addSubview(pickerView)
+        pickerView.isHidden = true
+        
+        previousButton.isEnabled = false
+        nextButton.isEnabled = false
```

Comment:
There's a bug with tapping on the search field when the picker is open. If you used the responder chain to present the picker view, this should clean up a bit. Instead of adding `pickerView` and `toolbar` directly to the view hierarchy, you can return them for the properties `inputView` and `inputAccessoryView`. Just have this view controller return `true` for `canBecomeFirstResponder`, and to present the picker call `self.becomeFirstResponder()`. I think it will work a bit more naturally, and will cover the tab bar and will automatically dismiss if the search field is pressed.